daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ledgerinteraction/GrpcLedgerClient.scala

```diff
@@ -116,6 +116,17 @@ class GrpcLedgerClient(val grpcClient: LedgerClient, val applicationId: Applicat
     }
   }
 
+  override def queryViewContractId(
+      parties: OneAnd[Set, Ref.Party],
+      interfaceId: Identifier,
+      cid: ContractId,
+  )(implicit
+      ec: ExecutionContext,
+      mat: Materializer,
+  ): Future[Option[Value]] = {
+    sys.error("not implemented") // TODO https://github.com/digital-asset/daml/issues/14830
```

Comment:
what breaks without this? what's missing to implement it?

Reply:
What breaks: Use of  `queryViewContractId` when connected to a _real_ ledger (over the `Grpc` ledger API).

Reply:
For example: if we add to the tests in: `daml-script/test/daml/TestInterfaces.daml`

Reply:
gotcha, thanks!